runtime/parser/parse_metrics_view.go

```diff
@@ -40,6 +40,11 @@ type MetricsViewYAML struct {
 		Ignore      bool   `yaml:"ignore"` // Deprecated
 		Unnest      bool
 		URI         string
+		Lookup      *struct {
```

Comment:
If you don't make this a pointer, the code gets simpler:
```suggestion
		Lookup      struct {
```